scripts/fast-reboot

```diff
@@ -663,6 +663,17 @@ if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
     fi
 fi
 
+TEAMD_INCREASE_RETRY_COUNT=0
+if [[ "${REBOOT_TYPE}" = "warm-reboot" || "${REBOOT_TYPE}" = "fastfast-reboot" ]]; then
+    /usr/local/bin/teamd_increase_retry_count.py --probe-only || TEAMD_RETRY_COUNT_PROBE_RC=$?
+    if [[ TEAMD_RETRY_COUNT_PROBE_RC -ne 0 ]]; then
+        debug "Warning: Retry count feature support unknown for one or more neighbor devices; assuming that it's not available"
+        TEAMD_INCREASE_RETRY_COUNT=0
```

Comment:
Are we supposed to bail out from here?

Reply:
For the images we deploy to our internal environment, this will be changed to bail out and fail the warm-reboot if SONiC neighbors are not seen. For the public version, since we can't require using SONiC-only neighbors for warm reboot, this will only be a warning.